client/components/hoc/protect-form/README.md

```diff
@@ -0,0 +1,34 @@
+Protect-Form
```

Comment:
I say this having introduced the `components/data` directory, but we're generally averse to nesting folders like this (`components/hoc`). Since a higher-order component is technically a function, not a component, do you think `lib` might be a better choice? Or at the very least `components` base directory.